dispatch/go.sum

```diff
@@ -0,0 +1,515 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
not sure whether we need this file or it can be generated in Dockrfile

Reply:
go.mod lists the dependencies and their versions.
go.sum is a security layer, it lists the checksum of the dependencies. This ensures that the dependency used at build time is the one wanted and not an imposter.

It is recommended that go.sum is checked in along with go.mod, see this [explanation](https://golangbyexample.com/go-mod-sum-module/)